DomainContent/NewYearBallDrop/ballDrop.js

```diff
@@ -0,0 +1,250 @@
+//
+// ballDrop.js
+// 
+// Created by Rebecca Stankus on 12/06/2018
+// Copyright High Fidelity 2018
+//
+// Licensed under the Apache 2.0 License
+// See accompanying license file or http://apache.org/
+//
+
+(function () {
+    var _this;
+
+    var TIME_CHECK_INTERVAL_MS = 50;
+    var FIREWORK_INTERVAL = 1000;
+    var ONE_THOUSAND = 1000;
+    var SECONDS_PER_MINUTE = 60;
+    var MINUTES_PER_HOUR = 60;
+    var TWO_DIGITS = 10;
+    var TARGET_TIME_INDEX = 13;
+    var BALL_START_POSITION = { x: -29.8177, y: 6.8299, z: -108.6157 };
+    var INTERVAL_DROP_DISTANCE_M = 0.0090;
+    var MUSIC = SoundCache.getSound(Script.resolvePath("assets/audio/auldJimi.mp3"));
+    var EXPLOSION = SoundCache.getSound(Script.resolvePath("assets/audio/explosion.mp3"));
+    var FIREWORK = SoundCache.getSound(Script.resolvePath("assets/audio/firework.mp3"));
+    var TICK = SoundCache.getSound(Script.resolvePath("assets/audio/tick.wav"));
+    var TOCK = SoundCache.getSound(Script.resolvePath("assets/audio/tock.wav"));
+    var HOURS_PER_DAY = 24;
+    var MAX_NUMBER_OF_DROPS = 27;
+
+    var injector;
+    var ballDropInterval;
+    var currentPosition = { x: -29.8177, y: 6.8299, z: -108.6157 };
+    var confetti;
+    var ball = "{6414271b-f0f4-4a62-b196-ac83dfd95253}";
```

Comment:
I'm not sure if you should be doing this because if the domain resets, all of those ids are going to change.  This should probably be done with something more like:
var entityID = Entities.findEntitiesByName("{Entity-Name}", MyAvatar.position, 10, false);